tests/lib/prepare.sh

```diff
@@ -145,5 +148,9 @@ prepare_all_snap() {
         fi
     fi
 
+    # ensure no test creates havoc
+    for name in pc pc-kernel ubuntu-core; do
+        snap list | grep $name
```

Comment:
I think we just need the "no havoc" comment to be echoed:

```
echo "Ensure fundamental snaps are still present
for ...
```
That would make the "snap list+grep" failure obvious.